README.md

```diff
@@ -5,13 +5,14 @@ Firmware designed for the pico of the IDC project
 
 # Building:
   
-1. create and enter desired build directory
-2. run `cmake ..`
-3. run `make`
+1. create your build directory `mkdir build`
+2. run `cmake -DPICO_BOARD=pico_w  -DWIFI_SSID='SSID' -DWIFI_PASSWORD='PASS' ..`
```

Comment:
This is not necessary. all those flags are already inside of the CMakeList file

Reply:
pico_w is inside CMakeLists, but SSID and PASSWORD isn't. In CMakeLists is specified to GRAB those secrets and use it on code

Reply:
You are correct, sorry. Nevertheless, I would include the lines

set(WIFI_SSID SSID)
set(WIFI_PASSWORD PASS)

into the CMakeList.txt file, for a cleaner cmake command

Reply:
I would not, due to the increased configuration complexity for the end user